WatchFaceKotlin/wear/src/main/java/com/example/android/wearable/watchfacekotlin/config/ColorSelectionRecyclerViewAdapter.kt

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.android.wearable.watchfacekotlin.config
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.support.wearable.view.CircledImageView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.core.content.edit
+
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.android.wearable.watchfacekotlin.R;
+
+import java.util.ArrayList;
```

Comment:
Is importing `ArrayList` really needed, the one from `kotlin.collections` is enough, right?
Also, why the semicolons on import lines in this Kotlin file?

Reply:
I'm not sure why the semi-colons were kept in some files... totally missed it too. Thanks for the catches. I've fixed it.

Reply:
Did you use J2K? Maybe it's a bug there to report on kotl.in/issue?